pkg/fetch/vendir.go

```diff
@@ -22,6 +23,11 @@ import (
 )
 
 const (
+	// GitURL source type to extract host
+	GitURL = iota
+	// ImageRegistry source type to extract host
+	ImageRegistry
+
 	vendirEntireDirPath = "."
 )
```

Comment:
Should we keep these separate?

```suggestion
	ImageRegistry
)

const (
	vendirEntireDirPath = "."
)
```